src/security/backups.md

```diff
@@ -0,0 +1,19 @@
+# Backup and restore
+
+Backups vary by our offering and their retention is governed by our [data retention](/security/data-retention.md) policies.
+
+Below are details on our Recovery Point Objective (RPO) and Recovery Time Objective (RTO) for our Platform Standard and Platform Enterprise offerings.
+
+## Platform Standard
```

Comment:
Platform Professional is how we refer to this.

Reply:
Please change "Platform Standard" to "Platform Professional" everywhere, in all documents.

https://platform.sh/pricing

Reply:
Pushed updated wording